packages/css/button.css

```
@@ -29,6 +29,11 @@
   text-align: inherit;
   text-decoration: none;
 
+  & svg,
+  & img {
+    min-width: 1em;
```

The icon might not be `1em`, so lets rather do this?
```suggestion
    flex-shrink: 0;
```

---

> The icon might not be 1em
What do you mean by this?

---

The reason the icon is shrinking, is because of `display: flex` on the parent button. `flex-shrink: 0` tells the browser that the icon should not shrink under its `flex-basis` size (which is `auto` and thus in this scenario pointing to the icons natural `width`). So, if the user scales up the icon with `<svg width=""` og `style={{ width: ` or simply use larger icons, all of this will be respected by `flex-shrink: 0`, instead of us implying a icon width for all consumers 🤝